### ISSUE TITLE
Internal server error on Export.get_active_time when task_id provided

### ISSUE THREAD

**Issue by andychisholm** [redbrick-ai/redbrick-sdk#166]
When calling [get_active_time](https://redbrick-sdk.readthedocs.io/en/stable/sdk.html#redbrick.export.Export.get_active_time) we can sucessfully retrieve a list in bulk, i.e.:

```
list(project.export.get_active_time(stage_name="Label"))

==

[{'orgId': '..',
  'projectId': '..',
  'stageName': 'Label',
  'taskId': '<task_id>',
  'completedBy': 'API Key',
  'timeSpent': 0,
  'completedAt': '2023-08-11T00:23:32.918137+00:00',
  'cycle': 0},
 {'orgId': '..',
  'projectId': '...',
  'stageName': 'Label',
  'taskId': '<another_task_id>',
  'completedBy': '..',
  'timeSpent': 4218,
  'completedAt': '2023-08-08T04:10:26.872142+00:00',
  'cycle': 0},
```

But if we try to list for a specific task:
```
list(
    project.export.get_active_time(
        stage_name="Label", task_id="<task_id>"
    )
)
```

We observe:
```
0 datapoints [00:00, ? datapoints/s]
[03:11:25] ERROR    INTERNAL SERVER ERROR                                                                          
0 datapoints [00:00, ? datapoints/s]
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ in <module>:1                                                                                    │
│                                                                                                  │
│ ❱ 1 list(                                                                                        │
│   2 │   p.export.get_active_time(                                                                │
│   3 │   │   stage_name="Label", task_id="<task_id>"                   │
│   4 │   )                                                                                        │
│                                                                                                  │
│ /home/coder/repos/report-svc/.venv/lib/python3.8/site-packages/redbrick/export/public.py:1250 in │
│ get_active_time                                                                                  │
│                                                                                                  │
│   1247 │   │   )                                                                                 │
│   1248 │   │                                                                                     │
│   1249 │   │   with tqdm.tqdm(my_iter, unit=" datapoints") as progress:                          │
│ ❱ 1250 │   │   │   for task in progress:                                                         │
│   1251 │   │   │   │   yield {                                                                   │
│   1252 │   │   │   │   │   "orgId": self.org_id,                                                 │
│   1253 │   │   │   │   │   "projectId": self.project_id,                                         │
│                                                                                                  │
│                                     ... 1 frames hidden ...                                      │
│                                                                                                  │
│ /home/coder/repos/report-svc/.venv/lib/python3.8/site-packages/redbrick/utils/pagination.py:54   │
│ in __next__                                                                                      │
│                                                                                                  │
│   51 │   │   │   # When no data is returned in the current iteration,                            │
│   52 │   │   │   # but there is still more data, go for the next iteration                       │
│   53 │   │   │   while True:                                                                     │
│ ❱ 54 │   │   │   │   self.datapoints_batch, self.cursor, *_ = self.func(                         │
│   55 │   │   │   │   │   max(0, min(self.concurrency, self.limit - self.total))                  │
│   56 │   │   │   │   │   if self.limit is not None                                               │
│   57 │   │   │   │   │   else self.concurrency,                                                  │
│                                                                                                  │
│ /home/coder/repos/report-svc/.venv/lib/python3.8/site-packages/redbrick/repo/export.py:415 in    │
│ active_time                                                                                      │
│                                                                                                  │
│   412 │   │   }                                                                                  │
│   413 │   │                                                                                      │
│   414 │   │   result = self.client.execute_query(query_string, query_variables, False)           │
│ ❱ 415 │   │   task_times = result.get("taskActiveTime", {}) or {}                                │
│   416 │   │   entries: List[Dict] = task_times.get("entries", []) or []                          │
│   417 │   │   return entries, task_times.get("cursor")                                           │
│   418                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by pritamrungta** [comment]
Hi @andychisholm 

Thank you for bringing this to our attention.
We've fixed this in the latest release [v2.13.6](https://pypi.org/project/redbrick-sdk/2.13.6/)
Could you please give it a try and let us know if it works?

Thanks!

**Comment by andychisholm** [comment]
Thanks @pritamrungta , confirmed fix with latest release.
